server/routes/return-of-service.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const keycloak = require('../keycloak');
+const logger = require('../logger.js');
+const { asyncMiddleware, applyMiddleware } = require('../error-handler.js');
+const { rosError } = require('../constants');
+const { CreateReturnOfServiceSchema, validate } = require('../validators');
+const { getParticipantByID } = require('../services/participants');
+const {
+  makeReturnOfServiceStatus,
+  getReturnOfServiceStatuses,
+} = require('../services/return-of-service');
+
+const router = express.Router();
+
+// Apply setup user middleware
+router.use(applyMiddleware(keycloak.setupUserMiddleware()));
+
+// Create a new return of service status
+router.post(
+  '/participant/:participantId',
+  applyMiddleware(keycloak.allowRolesMiddleware('health_authority', 'employer')),
+  asyncMiddleware(async (req, res) => {
+    const { participantId } = req.params;
+    const { email, user_id: userId, sub: localUserId } = req.user;
+    const user = userId || localUserId;
+    if (!(email && user)) {
+      return res.status(401).send('Unauthorized user');
+    }
+    // Validate body
+    await validate(CreateReturnOfServiceSchema, req.body);
+    // Validate Participant
+    const [participant] = await getParticipantByID({ id: participantId });
+    if (!participant) {
+      logger.error({
+        action: 'ros-status-create',
+        message: `Participant ${participantId} not found`,
+      });
+      return res.status(404).send('Participant not found');
+    }
+    const { data, status } = req.body;
+    try {
+      const response = await makeReturnOfServiceStatus({
+        participantId: participant.id,
+        data,
+        status,
+      });
+      logger.info({
+        action: 'ros-status-create',
+        performed_by: user,
+        id: response.id,
+      });
+      return res.status(201).json(response);
+    } catch (error) {
+      logger.error({
+        action: 'ros-status-create',
+        error: error.message,
+      });
+      switch (error.message) {
+        case rosError.participantNotHired:
```

Comment:
Why don't we define rosError as real messages so that we could return like this?
```
switch (error.message) {
  case rosError.participantNotHired:
  case rosError.noSiteAttached:
    return res.status(400).send(error.message);
```